src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplate.java

```diff
@@ -410,7 +411,13 @@ private DocumentCollection getDocCollection(String collectionName) {
             }
 
             queryStr += "r." + fieldName + "=@" + entry.getKey();
-            parameterCollection.add(new SqlParameter("@" + entry.getKey(), entry.getValue()));
+
```

Comment:
Can we wrap this for less details here? like add(xx.getKey, object). If there is another class Date_1 in future, there may be another branch in else-if. I think we can avoid the code like that.